packages/data-explorer/src/components/DataView/EntityTableRow.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <tr>
+    <td v-if="isShop"><shopping-button :isSelected="isSelected" @click.native="clickShop(id)"></shopping-button></td>
+    <td v-for="(item, index) in data" :key="index">{{item}}</td>
+  </tr>
+</template>
+
+<script>
+import ShoppingButton from '../Utils/ShoppingButton'
+export default {
+  name: 'EntityTableRow',
+  props: {
+    id: {
+      type: String,
+      required: true
+    },
+    data: {
```

Comment:
data is very general , suggest to use 2 word combination for more descriptive  prop